File: EVCare_BackEnd/DataAccess/Dtos/Appointment/AppointmentViewModel.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using DataAccess.Enums;

namespace DataAccess.Dtos.Appointment
{
    public class AppointmentViewModel
    {
        public int Id { get; set; }
        public DateTime AppointmentDate { get; set; }

        public string VehicleName { get; set; }
        public List<string> Services { get; set; }
        public string VehicleImageUrl { get; set; }
        public AppointmentStatusEnum Status { get; set; }
    }
}

File: EVCare_FrontEnd/src/models/AppointmentsModel/AppointmentViewModel.ts
import type { Dayjs } from "dayjs";
import type { AppointmentStatusEnum } from "../enums";

export type AppointmentViewModel = {
  id: number;
  appointmentDate: Dayjs;
  vehicleModel: string;
  customerName: string;
  phoneNumber: string;
  licensePlate: string;
  services: string[];
  vehicleImageUrl?: string[];
  status: AppointmentStatusEnum;
  note?: string;
};

//này có trong Staff_Appointment_Model
